### DISCUSSION TITLE
월드 안에서 퀘스트 수행에 대한 안내가 가능한가요?

### DISCUSSION THREAD

**Discussion by hbbng** [naverz/zepeto-studio-kor#234] [5. Q&A]
월드 안에서 게임이나 퀘스트를 수행하도록 제작하고 싶은데, 아래 기능들을 유니티로 직접 구현해서 제페토 월드에 적용할 수 있는지 궁금합니다.

 (1) 맵 입장시 퀘스트 안내에 대해 공지창이 뜨게 하는 것
 (2) 퀘스트 수행 횟수 자동 카운팅 하는 것
 (3) 모든 퀘스트를 완료했을때 최종 성공 여부를 안내 하는 것
    + 외부 웹페이지로 통하는 링크를 제공하는 것 (기프티콘 등의 경품 지급을 위해)
       동영상은 외부 링크 삽입이 안되는 걸로 알고 있는데, 단순 웹페이지로 빠지는것은 가능한지 궁금합니다.

(1)번 내용 예시
![Screenshot_20220121-112457_ZEPETO](https://user-images.githubusercontent.com/98136788/150455031-edb4411a-f9f5-490b-9874-2ee53360cd17.jpg)

(2)번 내용 예시
![count](https://user-images.githubusercontent.com/98136788/150455178-852ed78f-1829-4f11-8485-a0512587d6f6.jpg)

알려주시면 감사하겠습니다 ㅠㅠ

**Comment by hyeonjin-jo** [comment]
안녕하세요.

1~3번까지의 내용 모두 직접 구현해서 적용하는 것이 가능합니다.

웹페이지 링크의 경우 현재는 정책적으로 제한하고 있습니다.

**Reply by hbbng** [reply]
확인 감사합니다~!! 큰 도움이 되었습니다!
